src/Symfony/Component/Yaml/Tests/InlineTest.php

```diff
@@ -202,7 +202,7 @@ public function testParseUnquotedScalarStartingWithReservedIndicator($indicator)
 
     public function getReservedIndicators()
     {
-        return array(array('@'), array('`'));
+        return array(array('@'), array('`'), array('|'), array('>'));
```

Comment:
Yes, they have actual meaning (by the way, we only support `|`IIRC, should we add support for `>` too?).

Actually, I was not sure if using a deprecation trigger here is the right solution or if we should treat this as a bug fix and throw a `ParseException` instead. What do you think?